etl/steps/data/garden/tuberculosis/2023-11-27/drug_resistance_surveillance.meta.yml

```diff
@@ -0,0 +1,236 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Tuberculosis
+        - Causes of Death
+  display:
+    numDecimalPlaces: 0
+tables:
+  drug_resistance_surveillance:
+    variables:
+      pulm_labconf_new:
```

Comment:
Maybe it's better to write tuberculosis instead of TB?

Reply:
Good point!